src/iscn/iscnId.test.ts

```diff
@@ -0,0 +1,53 @@
+import { ISCNRecordData } from '../types';
+import { getMsgCreateIscnRecordJSON, getISCNIdPrefix } from './iscnId';
+
+const dummyMsgCreateIscnRecord :{
+  sender: string;
+  iscnRecord: ISCNRecordData;
+} = {
+  sender: 'like1ukmjl5s6pnw2txkvz2hd2n0f6dulw34h9rw5zn',
+  iscnRecord: {
+    recordNotes: '',
+    stakeholders: [],
```

Comment:
Do we want to add test case that does not contain optional fields like `stakeholders`?

Reply:
Since `ISCNRecordData` ([here](https://github.com/likecoin/iscn-js/blob/master/src/types.ts#L4-L9)) requires all fields, so I guess we cannot omit optional fields here.